### PR TITLE
PLATFORM-1955 get rid of Wikia::log - take #1

### PR DESCRIPTION
**Pull request by macbre**
[PLATFORM-1955](https://wikia-inc.atlassian.net/browse/PLATFORM-1955)

Get rid of `Wikia::log` and use proper severity levels (and include helpful context) when logging errors from:
- `LyrciFindTrackingService` (pinging @SeanColombo o here for code review),
- `SassService` and `AssetsManager`

Bonus points for adding our extra response headers (`X-Served-By` and `X-Backend-Response-Time`) to every AssetsManager's response (including HTTP 404).

@wladekb / @pchojnacki

### CODE REVIEW

File: extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ class LyricFindTrackingService extends WikiaService {
 	private function markLyricForRemoval($pageId) {
 		$this->wf->SetWikiaPageProp(WPP_LYRICFIND_MARKED_FOR_REMOVAL, $pageId, 1);
 
-		self::log(__METHOD__, "marked page #{$pageId} for removal");
+		$this->info( __METHOD__, [ 'page_id' => $pageId ] );
```
Comment (wladekb) [inline extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php]: I'd go with something more like `__CLASS__ - marked page ....` so the text messages would be meaningful (`__CLASS__` or `__METHOD__` - whatever you think fits better)

Comment (wladekb) [inline extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php]: This applies to a few more log statements below.

**Comment by wladekb** [comment]
generally lgtm, left 1 comment

File: extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php (hunk 1 above)
Comment (macbre) [inline extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php]: Addressed in a86e1c0 and e3091fe

**Comment by wladekb** [comment]
lgtm

**Pull request merged**